src/librbd/cache/pwl/LogOperation.cc

```diff
@@ -269,9 +266,9 @@ WriteLogOperationSet::~WriteLogOperationSet() { }
 
 std::ostream &operator<<(std::ostream &os,
                          const WriteLogOperationSet &s) {
-  os << "cell=" << (void*)s.cell << ", "
-     << "extent_ops_appending=[" << s.extent_ops_appending << ", "
-     << "extent_ops_persist=[" << s.extent_ops_persist << "]";
+  os << " cell=" << (void*)s.cell
+     << ", extent_ops_appending=[" << s.extent_ops_appending
+     << "], extent_ops_persist=[" << s.extent_ops_persist << "]";
```

Comment:
Did you check the resulting log message?  `extent_ops_appending` and `extent_ops_persist` are `C_Gather *`.  If it is printed as a regular pointer you could just remove the brackets.